.github/workflows/auto-merge-deps.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow will auto merge a PR authored by dependabot[bot]. It runs only on open PRs ready for
+# review.
+#
+# It will merge the PR only if: it is authored by dependabot[bot], is a patch semantic update, and
+# all CI checks are successful (ignoring the soon-to-be-removed Jenkins check).
+#
+# The workflow is divided into multiple sequential jobs to allow giving only minimal permissions to
+# the GitHub token passed around.
+#
+# Once we're using the merge queue feature, I think we can simplify this workflow a lot by relying
+# on dependabot merging PRs via its commands, as it will always wait for checks to be green before
+# merging.
+name: Dependabot auto-merge updates
+on:
+  pull_request:
+    types:
+      - ready_for_review
+      - opened
+      - reopened
+      - synchronize
+
+permissions:
+  checks: read
+  pull-requests: write
+
+jobs:
+  auto-merge:
+    name: Auto-merge dependabot PRs
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'camunda/zeebe' && github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: metadata
+        name: Fetch dependency metadata
+        if: false
```

Comment:
❓ `if: false`, so this step is always skipped?

Reply:
No, that's a left over for testing, my bad :see_no_evil: